### PR TITLE
[23] Debian bullseye support

### LINKED ISSUE #23: [FEATURE] Debian11 support
### Description

[Description of the issue]
Include Debian 11 support
### Why is this needed?

[A clear and concise description of why do you think this is needed]

### Additional Information

[Any additional information, configuration or data.]

### PR DESCRIPTION
**Pull request by xtianae7**
### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* All new code requires tests to ensure against regressions
* Remember to set **idealista:develop** as base branch;

### Description of the Change

Debian bullseye support

### Benefits

Role will be functional for debian bullseye
### Possible Drawbacks

None AFAIK
### Applicable Issues

Closes #23

### CODE REVIEW

**Review by blalop — APPROVED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -9,6 +9,7 @@ env:
   jobs:
     - MOLECULE_DISTRO=debian:stretch-slim
```
Comment (vicsufer) [inline .travis.yml:10]: When adding bullseye to test matrix we're removing stretch from it.

**Review by vicsufer — APPROVED** [review]

File: molecule/fpm/Dockerfile.j2 [hunk 2]
```diff
@@ -5,7 +5,14 @@ FROM {{ item.registry.url }}/{{ item.image }}
 {% else %}
 FROM {{ item.image }}
 {% endif %}
+
 # install minimal packages for debian slim images
+{% if item.image == 'debian:bullseye-slim' %}
+RUN apt-get update && \
+    apt-get install -y python3 sudo bash ca-certificates iproute2 systemd systemd-sysv python3-pip && \
+    apt-get clean
+{% else %}
```
Comment (frantsao) [inline molecule/fpm/Dockerfile.j2:14]: I would change these conditions:
if OS-> Debian Stretch then install python2 packages
else
install python3 packages.
I think it's better with a view to the moment when we will drop Debian Stretch support.

File: .travis.yml (hunk 1 above)
Comment (frantsao) [reply .travis.yml:10]: We need to maintain for a while support for Debian Stretch :-|

File: molecule/fpm/Dockerfile.j2 (hunk 2 above)
Comment (vicsufer) [reply molecule/fpm/Dockerfile.j2:14]: But buster still uses python2, rigth?

Comment (frantsao) [reply molecule/fpm/Dockerfile.j2:14]: Nope, you can use it, but you can survive with only python3 (see mysql_role PR #80).

File: .travis.yml (hunk 1 above)
Comment (xtianae7) [reply .travis.yml]: We reached the conclusion to terminate the support given to Stretch in this role. Debian buster support should be enough. Stretch and buster are very similar so we can assume that if the role works for Buster, it should work for Stretch.

File: molecule/fpm/Dockerfile.j2 (hunk 2 above)
Comment (xtianae7) [reply molecule/fpm/Dockerfile.j2:14]: In the end, we will keep it this way in this role. It is true that we could survive only with python3, but Debian buster uses python2 by default.

**Pull request merged**